bundles/org.smarthomej.binding.viessmann/src/main/java/org/smarthomej/binding/viessmann/internal/dto/features/FeatureDefaultSetterParamsDouble.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright (c) 2021-2022 Contributors to the SmartHome/J project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.smarthomej.binding.viessmann.internal.dto.features;
+
+/**
+ * The {@link FeatureDefaultSetterParamsDouble} is responsible for
```

Comment:
Please add a proper class description (also in other places).
```suggestion
 * The {@link FeatureDefaultSetterParamsDouble} provides default parameters of double features
```